std_msgs/Header header
float64 h_reader             # h_reader reading
float64 h_reader_variance    # h_reader_variance reading
